### PR TITLE
Update german translations

### PR DESCRIPTION
**Pull request by kripton**

### CODE REVIEW

File: webaccess/src/webaccess_de_DE.ts [hunk 1]
```diff
@@ -16,23 +16,23 @@
     <message>
         <location filename="webaccess.cpp" line="396"/>
         <source>Username and password are required fields.</source>
-        <translation type="unfinished"></translation>
+        <translation>Benutername und Passwort sind benötigte Felder.</translation>
```
Comment (janosvitok) [inline webaccess/src/webaccess_de_DE.ts]: Z is missing here. "Benutzername"

File: webaccess/src/webaccess_de_DE.ts [hunk 2]
```diff
@@ -116,83 +116,83 @@
...
         <location filename="webaccessconfiguration.cpp" line="287"/>
         <source>New password...</source>
-        <translation type="unfinished"></translation>
+        <translation>Neues Passwort...</translation>
     </message>
     <message>
         <location filename="webaccessconfiguration.cpp" line="286"/>
         <source>Username and password are required fields.</source>
-        <translation type="unfinished"></translation>
+        <translation>Benutername und Passwort sind benötigte Felder.</translation>
```
Comment (janosvitok) [inline webaccess/src/webaccess_de_DE.ts]: Here again, BenutZername

**Comment by kripton** [comment]
@janosvitok You're absolutely right, thanks for reviewing, fixing and merging!

**Comment by janosvitok** [comment]
You're welcome :) Thanks for writing it in the first place!

**Pull request merged**